### DISCUSSION TITLE
Renovate not updating existing renvovate PR

### DISCUSSION THREAD

**Discussion by Rahul-learn** [renovatebot/renovate#9898] [Request Help] [answered]
**Renovate type**

Self hosted

**Renovate platform**

GitHub Enterprise

**Renovate version (if self hosting)**

I'm self hosting Renovate with version: 24.32.2

**Relevant debug logs**

This is to follow up on the issue #9802

We are seeing below error in renovate logs

`Error committing files (repository=***/***)
       "err": {
         "task": {
           "commands": [
             "push",
             "origin",
             "renovate/***/**",
             "--force",
             "-u",
             "--no-verify",
             "--verbose",
             "--porcelain"
           ],
           "format": "utf-8"
         },
         "message": "Pushing to https://github.***.git\nPOST git-receive-pack (1097 bytes)\nremote: error: GH003: Sorry, force-pushing to renovate/**** is not allowed.        \nerror: failed to push some refs to 'https://github.***.git'\n",
         "stack": "Error: Pushing to https://github.***.git\nPOST git-receive-pack (1097 bytes)\nremote: error: GH003: Sorry, force-pushing to renovate/com.**** is not allowed.        \nerror: failed to push some refs to 'https://github.***.git'\n\n    at GitExecutorChain.onFatalException (/usr/src/app/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:67:77)\n    at GitExecutorChain.<anonymous> (/usr/src/app/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:59:21)\n    at Generator.throw (<anonymous>)\n    at rejected (/usr/src/app/node_modules/simple-git/src/lib/runners/git-executor-chain.js:6:65)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"
       }`

Branch protection rule is not enabled for these repos. 

May I know why renovate is doing force push here ?

Note: Renovate is able to create new PR but failing to update the existing PR.

**Comment by viceice** [comment, marked as answer]
Renovate will recreate the commit when branch needs an update. So we need to force push to update an existing branch / pr.

**Reply by viceice** [reply]
So you need to allow force push for renovate.

**Reply by Rahul-learn** [reply]
We are seeing this error recently and we have force push disabled always. May I know this is added in renovate latest version ?

**Reply by viceice** [reply]
Nope, this was always the same behavior. If renovate did update the commit, it recreate and force push.

**Reply by Rahul-learn** [reply]
Do we have any external settings available to change the force push to normal push? since we have force push disabled in enterprise wide renovate is failing in this case.

**Reply by rarkins** [reply]
No
